package.json

```diff
@@ -1,24 +1,50 @@
 {
-  "babel": {
-    "presets": [
-      "es2015"
-    ]
-  },
   "dependencies": {
+    "class-set": "^0.0.4",
+    "codemirror": "^5.39.2",
     "is-valid-email": "0.0.2",
+    "react": "^16.4.2",
+    "react-codemirror2": "^5.1.0",
+    "react-dom": "^16.4.2",
+    "react-router-dom": "^4.3.1",
     "url": "^0.11.0",
-    "valid-url": "^1.0.9",
-    "virtual-dom": "^2.1.1"
+    "valid-url": "^1.0.9"
+  },
+  "browserify": {
+    "transform": [
+      [
+        "babelify",
+        {
+          "sourceMaps": true,
+          "sourceMapsAbsolute": true
+        }
+      ]
+    ]
   },
   "devDependencies": {
-    "babel-core": "^6.6.5",
-    "babel-preset-es2015": "^6.6.0",
-    "babelify": "^7.2.0",
-    "browserify": "^13.0.0"
+    "@babel/core": "^7.0.0-rc.1",
+    "@babel/preset-env": "^7.0.0-rc.1",
+    "@babel/preset-react": "^7.0.0-rc.1",
+    "babelify": "^9.0.0",
+    "browserify": "^16.2.2",
+    "envify": "^4.1.0",
+    "prettier": "1.14.2",
+    "uglify-js": "^3.4.7",
+    "uglifyify": "^5.0.1"
+  },
+  "prettier": {
+    "semi": false,
+    "arrowParens": "avoid",
+    "insertPragma": true,
+    "printWidth": 80,
+    "proseWrap": "preserve",
+    "singleQuote": true,
+    "trailingComma": "none",
+    "useTabs": false,
+    "jsxBracketSameLine": false,
+    "bracketSpacing": false
   },
   "scripts": {
-    "build-js": "browserify formspree/static/js/main.js -t babelify --outfile formspree/static/js/bundle.js",
-    "build-css": "scss --sourcemap=none formspree/static/scss/main.scss formspree/static/css/main.css",
-    "build": "npm run build-js && npm run build-css"
+    "heroku-postbuild": "make prod"
```

Comment:
I love me some makefiles, but uh, NPM is actually pretty good at making stuff. Can we just use it rather than introducing yet another build system? 

https://www.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/

Reply:
We can, but I think the Makefile it is way simpler, the dependency tracking can get quite confusing with just npm scripts.

Normal people would be using gulp.